docs/docs/webchat/webchat-connectivity.md

```diff
@@ -0,0 +1,53 @@
+---
+id: webchat-connectivity
+title: Connectivity
+---
+
+Sometimes, the connection can be lost between the server and the chatbot. To handle this situation, you can set specific parameters to quickly detect the connection issue.
+
+For a better user experience, you can also warn the user by configuring a customized error message to be displayed within the webchat interface.
+
+To do so:
```

Comment:
To do so, you can configure the server options to tune timings for resolving Webchat's connection.